src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -14,16 +14,18 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+
+#import "MTRBaseDevice_Internal.h"
+
+#import <Matter/MTRCluster.h>
 #import <Matter/MTRClusterConstants.h>
-#import <Matter/MTRDefines.h>
```

Comment:
Is the assumption that all the SDK headers include this one?

Reply:
No, simply because there's no need for `Foo.mm` to repeat imports from its counterparts (`Foo.h` and `Foo_Internal.h`). Similarly, `Foo_Internal.h` should always import `Foo.h` as it's first line, so no repeating of anything from `Foo.h` needed there either.

Reply:
Ah, so the key is that MTRBaseDevice.h imports MTRDefines?